wdl2cwl/tests/cwl_files/blast.cwl

```diff
@@ -0,0 +1,335 @@
+cwlVersion: v1.2
+$graph:
+- class: Workflow
+  id: main
+  inputs:
+    - id: blast_docker_override
+      type:
+        - string
+        - 'null'
+    - id: blast_docker
+      type: string
+      # TODO: how to translate the original?
+      #       String blast_docker = select_first([blast_docker_override,"swr.cn-south-1.myhuaweicloud.com/cngbdb/blast:1.2"])
+    - id: queryfa
+      type: File
+    - id: fname
+      type: string
+      default: '/sfs/blastdb/2019_ncov/nucl/v6/ncov'
+    - id: method
+      type: string
+      default: 'blastn'
+    - id: outfmt
+      type: int
+      default: 7
+    - id: evalue
+      type: float
+      default: 10
+    # TODO: how to translate the original?
+    #       String Outfile = basename(queryfa)+'.blast_result.txt'
+    - id: Outfile
+      type: string
```

Comment:
:point_up:

Reply:
This is another WDL expression in the definition of an input; I've made an issue for that https://github.com/common-workflow-lab/wdl-cwl-translator/issues/240